recaptcha_enterprise/snippets/test_create_assessment.py

```diff
@@ -137,3 +159,21 @@ def assess_token(recaptcha_site_key: str, token: str, action: str) -> Assessment
 
 def set_score(browser: WebDriver, score: str) -> None:
     browser.find_element(By.CSS_SELECTOR, "#assessment").send_keys(score)
+
+
+def get_hashed_account_id() -> str:
+    account_id = "alicebob"
+    key = "your_secret_key"
+
+    salt = secrets.token_hex(16)
+    salted_account_id = salt + account_id
+
+    # Encode the key and salted message as bytes
+    key_bytes = bytes(key, 'utf-8')
+    salted_message_bytes = bytes(salted_account_id, 'utf-8')
+
+    # Create an HMAC SHA-256 hash of the salted message using the key
+    hashed = hmac.new(key_bytes, salted_message_bytes, hashlib.sha256)
+
+    # Get the hex-encoded digest of the hash
+    return hashed.hexdigest()
```

Comment:
nit: if this is only used by `test_mfa_assessment`, could you move this above line 115? If this is going to be used by other methods / other tests, refactoring into a util class will also work.

Reply:
I've refactored it to a util class, since I suspect more methods might be using this.